llvm/test/CodeGen/AArch64/arm64-named-reg-alloc.ll

```diff
@@ -4,7 +4,7 @@
 define i32 @get_stack() nounwind {
 entry:
 ; FIXME: Include an allocatable-specific error message
```

Comment:
Essentially, the bug is what this FIXME says. The error message is, as we say here "rubbish" :)